src/components/ProjectField/Meta.tsx

```diff
@@ -40,7 +40,7 @@ const ProjectFieldMeta = ({ date, dateLabel, userId, userName, userLabel }: Proj
               marginBottom={theme.spacing(1)}
               component={'span'}
             >
-              {getDateDisplayValue(date || '')}
+              {date ? getDateDisplayValue(date) : ''}
```

Comment:
I suppose this fixes https://terraformation.atlassian.net/browse/SW-5173

Reply:
I believe so, it was a race condition so somewhat hard to reproduce.